seqr/views/utils/permissions_utils.py

```diff
@@ -51,16 +56,15 @@ def login_and_policies_required(view_func):
 
 def _user_has_policies_and_passes_test(test_func):
     def decorator(view_func):
-        return _active_required(_current_policies_required(user_passes_test(test_func, login_url=API_LOGIN_REQUIRED_URL)(view_func)))
+        return login_and_policies_required(user_passes_test(_require_permission(test_func))(view_func))
```

Comment:
So thoughts as above, `test_func` is better to be renamed as `permission_test_func`.